bitnami/kubernetes-event-exporter/templates/deployment.yaml

```diff
@@ -26,9 +26,11 @@ spec:
     metadata:
       annotations:
         checksum/config: {{ include "kubernetes-event-exporter.config" . | sha256sum }}
-        {{- if or .Values.podAnnotations .Values.commonAnnotations }}
-        {{- $podAnnotations := include "common.tplvalues.merge" ( dict "values" ( list .Values.podAnnotations .Values.commonAnnotations ) "context" . ) }}
-        {{- include "common.tplvalues.render" ( dict "value" $podAnnotations "context" $ ) | nindent 8 }}
+        {{- if .Values.podAnnotations }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.podAnnotations "context" $) | nindent 8 }}
+        {{- end }}
+        {{- if .Values.commonAnnotations }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.commonAnnotations "context" $) | nindent 8 }}
```

Comment:
The problem with this approach is that it doesn't protect the chart from including duplicated annotations (e.g. if the same annotation key is used both in `podAnnotations` and `commonAnnotations` parameters)

Reply:
Try using:
```yaml
podAnnotations:
  vault.hashicorp.com/agent-inject-template-password: |
    {{ printf "{{`{{- with secret \"kubernetes-secrets/data/tooling/es-node/k8s_events_exporter\" }}\n  export ES_USERNAME=\"k8s_events_exporter\"\n  export ES_PASSWORD=\"{{ .Data.data.k8s_events_exporter }}\"\n  {{- end }}" }}`}}" }}
```